mne_bids/mne_bids.py

```diff
@@ -13,6 +13,7 @@
 from collections import defaultdict, OrderedDict
 
 import numpy as np
+from mne import Epochs
```

Comment:
good point. This is also something that should be discussed in the main spec, where we have the field `RecordingType` with possible values "continuous" and "epoched" ... this might be obsolete if epochs are considered derivatives

Reply:
I guess we can leave it in there. It's up to the user to ensure if the data they provide is "raw" or not ...